apps/web/data/curriculum.ts

```diff
@@ -4,92 +4,94 @@ export const BELTS: IBelt[] = [
+      "Após o primeiro cinturão, procura-se aprimorar os primeiros conceitos que foram introduzidos ao Ninja nas sessões anteriores, de forma a que se sinta mais confortável em usar o Scratch e comece já a criar projetos um pouco mais elaborados.",
     image: "yellow.png",
     requirements: [
-      "Maior autonomia com os conceitos básicos",
+      "Ter maior autonomia com os conceitos básicos",
       "Saber aplicar condicionais e ciclos",
       "Ter concluído pelo menos um projeto em Scratch",
     ],
   },
   {
     colour: EBelt.Blue,
     description:
-      "Para obter o cinturão Azul, o Ninja já deve possuir uma maior autonomia com o Scratch, assim como ter concluído outros 2 projetos.",
+      "Para obter o cinturão azul, o Ninja já deve possuir uma maior autonomia com o Scratch, assim como ter concluído outros 2 projetos. Deve também ter completado todos os níveis do Lightbot.",
     image: "blue.png",
     requirements: [
-      "Manipular as variáveis da forma correta",
-      "Apresentar um projeto a um Mentor",
+      "Saber manipular variáveis da forma correta",
+      "Saber aplicar o conceito de módulos",
+      "Completar todos os níveis do Lightbot",
       "Terminar outros 2 projetos em Scratch",
+      "Terminar pelo menos 1 projeto usando o conceito de módulos",
     ],
   },
   {
     colour: EBelt.Green,
     description:
-      "O cinturão Verde poderá ser obtido depois de um período de transição entre a plataforma Scratch e uma linguagem de programação mais textual, que ficará a critério do Mentor e do Ninja. Recomendam-se linguagens de fácil aprendizagem e compreensão como Python, por exemplo.",
+      "O cinturão verde poderá ser obtido depois de um período de transição entre a plataforma Scratch e uma linguagem de programação mais textual, que ficará a critério do Mentor e do Ninja. Recomendam-se linguagens de fácil aprendizagem e compreensão como Python, por exemplo.",
     image: "green.png",
     requirements: [
       "Aplicar o que aprendeu no Scratch noutra linguagem",
-      "Compreender os conceitos e as aplicações de Listas e Strings",
+      "Compreender os conceitos e as aplicações de listas e strings",
       "Terminar 2 projetos na nova linguagem (1 deles pode ser uma adaptação de algum já feito em Scratch, se o Ninja quiser)",
-      "Apresentar um projeto a 2 mentores",
     ],
   },
   {
     colour: EBelt.Orange,
     description:
-      "Com o cinturão Laranja, o Ninja aprenderá a utilizar ferramentas da sua linguagem escolhida para criar interfaces ou até jogos simples. O Ninja deverá aprender também sobre os conceitos de I/O, com recurso às bibliotecas do sistema operativo.",
+      "Com o cinturão laranja, o ninja aprenderá a utilizar ferramentas da sua linguagem escolhida para criar interfaces ou até jogos simples. O Ninja deverá aprender também HTML, de modo a conseguir criar uma página web simples.",
```

Comment:
There are places where we use "Ninja" and places where we use "ninja" (downcase). I think there should be some sort of coherence. 🙏🏼

Reply:
You're right! We missed that one.